src/commons/validation.service.ts

```diff
@@ -0,0 +1,18 @@
+import { Injectable } from '@nestjs/common';
+import { CalendarEventAuthor } from '../calendar/interfaces/calendar.interface';
+import { DiscordProfileAuthor } from '../discord/interfaces/discord.interface';
+import { StandupAuthor } from '../standup/interfaces/standup.interface';
+
+@Injectable()
+export class ValidationService {
+  public validateAuthor(
+    objectAuthor: CalendarEventAuthor | DiscordProfileAuthor | StandupAuthor,
+    headerUid: string | string[],
+    headerPlatform: string | string[],
+  ) {
+    if (objectAuthor.uid !== headerUid) return false;
+    if (objectAuthor.platform !== headerPlatform) return false;
+
+    return true;
+  }
+}
```

Comment:
Would this be better in the `auth` module?

Reply:
Probably yes, I did this in case we need to validate other things in the future as well